### ISSUE TITLE
Under Armour API URL change! Nov 15 deadline!

### ISSUE THREAD

**Issue by MrCsabaToth** [TrackMyIndoorWorkout/TrackMyIndoorWorkout#523]
TL;DR
API URLs are changing and a new API Terms of Use needs to be accepted before Nov. 15th.

Dear Partner,

We hope this message finds you well. We are excited to inform you that MapMyFitness has been acquired by Outside Interactive, Inc., a move that will strengthen our combined mission of helping athletes perform better through passion, design, and innovation.

Our platform powers thousands of partners and developers across our connected fitness applications – MapMyFitness, MapMyRun, MapMyRide, and MapMyWalk. With this acquisition, we remain committed to delivering the same high-quality platform you know and trust:

* Turn-key and state-of-the-art solutions
* Optimized for insights over information
*Driven by social experiences that inspire athletic performance

As part of this transition, we are bringing your attention to an important update that will impact your integration with our platform.

We are updating our API URL. The current URL schema will be supported until November 15, 2024. 

To ensure continued functionality of your applications and services, you will need to update your API endpoints:

* Current: https://api.ua.com
* New: https://api.mapmyfitness.com

While we have implemented redirects, please make sure to update your API requests accordingly.

We recommend making these changes by November 15, 2024 to avoid any disruption.

In addition, we are now providing updated login button images, which can be found in the Authentication Overview portion of our API Documentation. If you have users connect to your application using their MapMyFitness accounts, you are required to update your login buttons to use the new images by November 15, 2024.

Finally, we have updated our Privacy Policy and Terms of Use to reflect this acquisition and to better serve our growing community. We encourage you to review these updated documents:
* Privacy Policy
* Terms of Use
* API Terms of Service

Furthermore, to continue using the MapMyFitness API, we need you to read and accept the updated Outside API Terms of Service by completing the form found here: https://www.surveymonkey.com/r/NPSPQ7W

Please Note: Partners who have not accepted the updated API Terms of Service using the valid email linked to their MapMyFitness Application (this email) will start having their accounts deactivated after November 15, 2024.

If you have any questions or need assistance, please don’t hesitate to reach out to our support team at partner-support@mapmyfitness.com or visit our updated API documentation at developer.mapmyfitness.com.

We look forward to continuing our collaboration as we enter this exciting new chapter.

Best regards,
Kevin Callahan
VP, Product @ Outside
Cofounder, MapMyFitness

**Comment by MrCsabaToth** [comment]
1. Icons downloaded from https://developer.mapmyfitness.com/docs/v70_Authentication/
2. png to SVG vectorization by https://picsvg.com/
3. SVG Optimization by https://svgomg.net

Color: #0D4C8B
